packages/account/package.json

```
@@ -35,6 +35,7 @@
         "@deriv-com/utils": "^0.0.25",
         "@deriv-com/ui": "^1.28.1",
         "@deriv/api": "^1.0.0",
+        "@deriv-com/quill-ui": "^1.12.3",
```

```suggestion
        "@deriv-com/quill-ui": "^1.12.4",
```
this is the last one :)

---

Okieee let me update it 😎